db/migrate/20220916093407_create_order.rb

```diff
@@ -0,0 +1,14 @@
+class CreateOrder < ActiveRecord::Migration[6.1]
+  def change
+    create_table :orders do |t|
+      t.integer :customer_id
+      t.decimle :total_price
+      t.integer :discount
```

Comment:
confusing names
discount and discounted_value,
discount -> discount_percentage
discounted_value -> discounted_value

Reply:
changed